plugins/serializers/influx/influx_test.go

```diff
@@ -323,6 +323,70 @@ var tests = []struct {
 		),
 		output: []byte("cpu,host=x\\ny value=42i 0\n"),
 	},
+	{
+		name: "empty tag value is removed",
+		input: MustMetric(
+			metric.New(
+				"cpu",
+				map[string]string{
+					"host": "",
+				},
+				map[string]interface{}{
+					"value": 42,
+				},
+				time.Unix(0, 0),
+			),
+		),
+		output: []byte("cpu value=42i 0\n"),
+	},
+	{
+		name: "empty tag key is removed",
+		input: MustMetric(
+			metric.New(
+				"cpu",
+				map[string]string{
+					"": "example.org",
```

Comment:
might also add a test for key `\` just to be thorough

Reply:
Fixed